### PR TITLE
 kubernetes-setup: compatibility with Ubuntu

### PR DESCRIPTION
**Pull request by MatthieuSarter**
* Adds compatibility with Ubuntu
* Makes installing kata-runtime optional

### CODE REVIEW

File: kubernetes-setup/main.yaml [hunk 1]
```diff
@@ -102,11 +91,33 @@
...
 
-    - name: Install CRI-O
-      dnf: 
+    - name: Install CRI-O on Fedora
+      dnf:
         name: cri-o
         state: present
+      when: ansible_distribution == "Fedora"
+
+    - name: Build & Install CRI-O on Ubuntu
```
Comment (bpradipt) [inline kubernetes-setup/main.yaml:102]: How about using the `kubic` repos for Ubuntu as mentioned here - https://cri-o.io/ ?

Comment (MatthieuSarter) [reply kubernetes-setup/main.yaml:102]: We could use them on x86-64, but those repos are not providing ppc64le binaries for Ubuntu (see http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.21/xUbuntu_20.04/ for example).

Comment (bpradipt) [reply kubernetes-setup/main.yaml:102]: Ah ok..

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @MatthieuSarter

**Pull request merged**